### PR TITLE
Jabera 6

### PR DESCRIPTION
**Pull request by jkk84**

### CODE REVIEW

File: src/main/java/com/the/dark/side/crew/fejsbuk/controller/CommentController.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/comments")
+public class CommentController {
+
+    private final CommentService commentService;
+
+    @GetMapping
+    public List<CommentDto> getAllPostIdComments(@RequestParam long postId) {
```
Comment (rmaj91) [inline src/main/java/com/the/dark/side/crew/fejsbuk/controller/CommentController.java]: getAllComments or getComments is enough

File: src/test/java/com/the/dark/side/crew/fejsbuk/mapper/CommentMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        when(commentDto.getUserId()).thenReturn(userId);
+        when(commentDto.getPostId()).thenReturn(postId);
+        when(userRepository.findById(userId)).thenReturn(Optional.of(userEntity));
+        when(postRepository.findById(postId)).thenReturn(Optional.of(postEntity));
+        when(userEntity.getId()).thenReturn(userId);
+        when(postEntity.getId()).thenReturn(postId);
+
+        CommentEntity result = commentMapper.toEntity(commentDto);
+        assertEquals(message, result.getMessage());
+        assertEquals(userId, result.getUserEntity().getId());
```
Comment (rmaj91) [inline src/test/java/com/the/dark/side/crew/fejsbuk/mapper/CommentMapperTest.java]: Just check if it's user which u expected

File: src/test/java/com/the/dark/side/crew/fejsbuk/mapper/CommentMapperTest.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        assertEquals(postId, result.getPostEntity().getId());
```
Comment (rmaj91) [inline src/test/java/com/the/dark/side/crew/fejsbuk/mapper/CommentMapperTest.java]: same

File: src/test/java/com/the/dark/side/crew/fejsbuk/service/impl/CommentServiceImplTest.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    void whenPostFoundThenReturnAllComments() {
+        long postId = 2L;
+        CommentDto commentDto = mock(CommentDto.class);
+        CommentEntity commentEntity = mock(CommentEntity.class);
+        PostEntity postEntity = mock(PostEntity.class);
+
+        when(commentRepository.findByPostEntityId(postId))
+                .thenReturn(List.of(commentEntity));
+        when(commentMapper.toDtos(List.of(commentEntity)))
+                .thenReturn(Collections.singletonList(commentDto));
```
Comment (rmaj91) [inline src/test/java/com/the/dark/side/crew/fejsbuk/service/impl/CommentServiceImplTest.java]: be consistent use List.of or singleton

File: src/test/java/com/the/dark/side/crew/fejsbuk/mapper/CommentMapperTest.java [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+        doReturn(commentDto).when(mapper).toDto(commentEntity);
+
+        List<CommentDto> result = mapper.toDtos(List.of(commentEntity));
+        assertEquals(1, result.size());
+        assertEquals(commentDto, result.get(0));
+    }
+
+    @Test
+    void whenCallToDtoThenReturnPostDto() {
```
Comment (rmaj91) [inline src/test/java/com/the/dark/side/crew/fejsbuk/mapper/CommentMapperTest.java]: just whenToDtoThenReturnPostDto, same in other tests

**Review by rmaj91 — COMMENTED** [review]
cool!

File: src/test/java/com/the/dark/side/crew/fejsbuk/mapper/CommentMapperTest.java (hunk 2 above)
Comment (jkk84) [reply src/test/java/com/the/dark/side/crew/fejsbuk/mapper/CommentMapperTest.java]: I don't understand, in PostMapperTest is the same situation and it's ok.

**Review by rmaj91 — APPROVED** [review]
cr ok.

**Pull request merged**